projects/uitdatabank/docs/search-api/advanced-queries.md

```diff
@@ -0,0 +1,1067 @@
+# Advanced queries
+
+With advanced queries you can make more complex queries, using boolean operators, search keywords and/or specific field values, and grouping.
+
+The syntax is based on the Lucene query syntax. More info about the syntax can be found in the [ElasticSearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html#query-string-syntax).
+
+## Supported fields
+
+### addressCountry
+
+With the `addressCountry` field you can limit your results to one or more countries.
+
+> By default, the search API will only return results that are in Belgium. In order to retrieve results outside Belgium you'll need to disable the default filter of the addressCountry URL parameter. You can reset this default as described in the [default filters guide](./default-filters.md).
+
+**Applicable on endpoints**
+
+`/events` `/places` `/offers` `/organizers`
+
+**Possible values**
+
+The value for this field must always be an [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) country code.
+
+**Examples**
+
+Because a document can have address translations, you can search by a specific language or alternatively use a wildcard instead of specifying a language, as in the example below. The query above will look for specific matches in the `nl` address:
+
+```
+GET /places/?q=address.nl.addressCountry:BE 
+```
+
+The following example looks for matches in any language:
+
+```
+GET /places/?q=address.\*.addressCountry:BE
+```
+
+### address.{language}.addressLocality
+
+With the `addressLocality` field you can limit your results to one or more municipalities.
```

Comment:
(Copied from older comment on previous filename)

Should we suggest to use the `postalCode` parameter instead when possible? Since the municipality names are free text in the API, so they may contain typo's or be incorrect.

Reply:
Clarified in 747b07e5031d75a266c8192ffb3dfe6d1fdd92fd I see 👍